assets/src/edit-story/components/form/color/colorPreview.js

```diff
@@ -105,7 +185,7 @@ function ColorPreview({ onChange, hasGradient, hasOpacity, value, label }) {
 }
 
 ColorPreview.propTypes = {
-  value: PatternPropType,
+  value: PropTypes.oneOfType([PatternPropType, PropTypes.string]),
```

Comment:
I'd still think it should be object value only. Right? Do we ever need to pass a string?

Reply:
[You made us need to](https://github.com/google/web-stories-wp/commit/4f3b29496567abefaed1fd7115a299946090e80b#diff-6309329f3ce58f72589cf14cc8f4a3feR62) 😉.

We could do `null` for `isMultiple` of course, but that would then revert to transparent, which is sometimes a valid value, other times not.

Reply:
Hmm. Maybe for another PR, but I think: `null` shouldn't be used (it means uncontrolled and we don't want to mix with that for now), empty string is a good value and could mean "transparent", MULTIPLE_VALUE means multiple value. These should be enough I think for majority of cases.